api/src/endpoints/svgThumbnail.js

```diff
@@ -19,20 +30,12 @@ routes.get('/:svgName', async (req, res) => {
     if (!fs.existsSync(svgFile)) {
       throw new Error(`2D map does not exist for ${svgName} in ${model}.`);
     }
-
-    const min_w = 100;
-    const max_w = 600;
-    var w_thumb = 400;
-    const isnum = /^\d+$/.test(width);
-    if (!isnum) {
-      throw new Error(`Invalid width provided: ${width}. It should be an integer between ${min_w} and ${max_w}.`);
-    } else {
-      w_thumb =  parseInt(width);
-      if (w_thumb < min_w || w_thumb > max_w){
-        throw new Error(`Invalid width provided: ${width}. It should be between ${min_w} and ${max_w}.`);
-      }
-    }
-    const svgThumbnail = await getSvgThumbnail(svgName, w_thumb, model)
+    const resizeParams = {
+      width: getDimension(width),
+      height: getDimension(height),
+      fit: 'inside',
```

Comment:
```suggestion
      fit: 'cover',
```
@e0 @nanjiangshu @inghylt what do you think?

Reply:
That is exactly what I was looking for and it solves the `and/or` thing I brought up before as well.

Reply:
If you specify both width and height, the map is cropped off for heights > 330, even if width is set to 600. But I'm not really sure of how the users will use this so if you feel like this is what we need, go for it

Reply:
To me this more of a back-up option, using `width` or `height` separately are not sufficient.